src/verifact_agents/claim_detector.py

```
@@ -1,104 +1,388 @@
+import asyncio
+from pydantic import BaseModel, Field
+from typing import List, Dict, Any
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)